style.css

```diff
@@ -0,0 +1,277 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+*::before, *::after {
+  box-sizing: inherit;
+}
+
+.boss {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  background-image: url(http://gaslampden.com/wp-content/uploads/sites/3/2014/04/surfer-1.jpg);
+
+  background-size: 100% 100%;
+  background-position: center;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header:before {
+  content: '';
+   position: absolute;
+   top: 0;
+   left: 0;
+   bottom: 0;
+   right: 0;
+   background: rgba(63, 76, 167, 0.7);
+
+}
+
+.header_navbar {
+  display:flex;
+  padding: 1rem;
+  padding-left: 15vw;
```

Comment:
Use a double container (`max-width` + `margin: auto`) to help line up this navbar in a way that will work at different screen sizes.